### PR TITLE
Propose quick fix pylint/typos

### PR DESCRIPTION
**Pull request by jeis2497052**
are these updates OK?

### CODE REVIEW

**Comment by gstein** [comment]
ezt.py is upstream. No patches allowed on that (create a PR at https://github.com/gstein/ezt) ... @Humbedooh is working on a patch to place ezt on all boxes, so that we can simply remove local copies from our modules.

The other changes generally look good. It would be great if you could back out the ezt change, and then we roll forward with the rest of your changs. Thanks!

**Comment by jeis2497052** [comment]
sorry about ezt, I will back that out shortly

File: modules/gitbox/files/matt/tools/grouper.py [hunk 1]
```diff
@@ -15,29 +15,30 @@
...
+                    format='[%(asctime)s]: %(message)s', level=logging.INFO)
 
 # LDAP Defs
 UID_RE = re.compile("uid=([^,]+),ou=people,dc=apache,dc=org")
 
 # Run `python grouper.py debug` to check teams but not add/remove users
 DEBUG_RUN = True if len(sys.argv) > 1 and sys.argv[1] == 'debug' else False
 if DEBUG_RUN:
-    print("Debug run active! Not modifying teams")
+    print "Debug run active! Not modifying teams"
```
Comment (gstein) [inline modules/gitbox/files/matt/tools/grouper.py]: Oh, I think that grouper.py might run under Py3, so this tweak would be a problem.

**Comment by gstein** [comment]
Thanks, @jeis2497052 ! ... very happy to see new faces contributing! Looks like a small tweak, and this can be merged.

Much appreciated!

**Comment by gstein** [comment]
@gmcdonald hashlib isn't used in the file, so removing that import is Good(tm) :-)

**Comment by jeis2497052** [comment]
Great!  so just add the parens for Py2 back then ?  sorry about the parens, I keep forgetting they are required in some versions Py2.

**Comment by gstein** [comment]
@jeis2497052 the parens are needed for Py3, as "print" switched from a statement to a function. For Py2, the parens are essentially a no-op.

Then we merge! :-)

**Comment by gstein** [comment]
Fabulous. Thanks, John! ... More, please :-)

**Comment by jeis2497052** [comment]
Happy to contribute.  Thanks for walking me through the upstream issues and paren issues.

**Pull request merged**